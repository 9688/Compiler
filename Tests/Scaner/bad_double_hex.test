p0x1
0x1p
0x1.p
.0x1p
0x1.0x1p*0x1
0x1.0x1E0x1