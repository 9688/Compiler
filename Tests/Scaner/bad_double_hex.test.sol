IDENTIFICATOR 1 1 p0x1
Ошибка в строке 2 позиции 4: требуется экспоненциальное значение
Ошибка в строке 3 позиции 5: требуется экспоненциальное значение
Ошибка в строке 4 позиции 5: требуется экспоненциальное значение
Ошибка в строке 5 позиции 8: требуется экспоненциальное значение
Ошибка в строке 6 позиции 9: неверная запись числа
EOF 6 12 
