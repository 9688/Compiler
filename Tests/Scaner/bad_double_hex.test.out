IDENTIFICATOR 1 1 p0x1
Ошибка в строке 2 позиции 4: требуется экспоненциальное значение
Ошибка в строке 3 позиции 5: требуется экспоненциальное значение
Ошибка в строке 4 позиции 5: требуется экспоненциальное значение
Ошибка в строке 5 позиции 8: требуется экспоненциальное значение
OPERATOR 5 9 *
DOUBLE 5 10 1
неверная запись числа
IDENTIFICATOR 6 10 x1
EOF 6 12 
